### ISSUE TITLE
Implement JSON Schema chai assertion

### ISSUE THREAD

**Issue by blikblum** [a85/POSTMan-Chrome-Extension#3582]
## App Details:

```
Postman for Windows
Version 5.2.1
win32 10.0.15063 / x64
```

## Feature request:

Implement JSON Schema chai assertion like `pm.response.to.match.jsonSchema` or `pm.response.to.match.jsonSchema`
 
The new [test API](https://www.getpostman.com/docs/postman/scripts/postman_sandbox_api_reference) comes with predefined chai assertions but is missing one for test if the response matches a JSON schema

There's already http://chaijs.com/plugins/chai-json-schema/ that could be integrated into Postman environment

**Comment by numaanashraf** [comment]
@blikblum Thanks for the suggestion. 😃

**Comment by kunagpal** [comment]
@blikblum This is being worked on in https://github.com/postmanlabs/postman-sandbox/pull/104

**Comment by codenirvana** [comment]
@blikblum We've added support for JSON schema assertions which uses `ajv` library under the hood.
This should be available in Postman native apps starting from 6.7.1.

```javascript
pm.response.to.have.jsonSchema(schema:Object)
pm.response.to.have.jsonSchema(schema:Object, ajvOptions:Object)
```

**Comment by vrolijken** [comment]
Is there any documentation on this? 

Why use another schema validator than the documented tv4? Can I use tv4.getSchema('somejsonpointer') as param?

**Comment by yokzy88** [comment]
Nevermind, 
Already found the solution:
```
pm.expect(JSON.parse(pm.request.body)).to.have.jsonSchema(schema:Object);
```

<del>Hi @codenirvana, 
Is this only work for pm.response?
I try this: 
```pm.request.to.have.jsonSchema(schema:Object);```
It return assertions error saying it has invalid schema
I also try this:
```pm.request.body.to.have.jsonSchema(schema:Object);```
and it return **Cannot read property 'have' of undefined**
</del>

> @blikblum We've added support for JSON schema assertions which uses `ajv` library under the hood.
> This should be available in Postman native apps starting from 6.7.1.
> 
> ```js
> pm.response.to.have.jsonSchema(schema:Object)
> pm.response.to.have.jsonSchema(schema:Object, ajvOptions:Object)
> ```
